spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -344,14 +342,10 @@ public TransportChannelProvider transportChannelProvider() {
 		return InstantiatingGrpcChannelProvider.newBuilder().build();
 	}
 
-	@Configuration
-	@ConditionalOnBean(ObjectMapper.class)
-	static class JacksonPubSubMessageConverterAutoConfiguration {
-
-		@Bean
-		@ConditionalOnMissingBean
-		public PubSubMessageConverter pubSubMessageConverter(ObjectMapper objectMapper) {
-			return new JacksonPubSubMessageConverter(objectMapper);
-		}
+	@Bean
+	@ConditionalOnMissingBean
+	public PubSubMessageConverter pubSubMessageConverter() {
+		return new SimplePubSubMessageConverter();
```

Comment:
I don't think we need to configure it at all, since it's used by default anyway.